01-introduction.md

```diff
@@ -74,11 +74,11 @@ so that they get the positive feedback of being able to read whole stories as ea
 The basic idea is that once children find reading rewarding,
 they'll be motivated to go back and figure out the letters.
 
-Both approaches are consistent with what we know about how children learn,
-so we need to study them in action to determine which is better.
-The findings have been contradictory:
-while some studies find that phonics works best,
-others show that whole language yields better results.
+These approaches are not equally consistent with what we know about how children
+learn. In fact, the scientific consensus today is that the whole word approach
+was a mistake. Nevertheless, when these methods were studied in classrooms, the
```

Comment:
Here's another more recent one (but it's behind a paywall, so I'm not
sure): http://psycnet.apa.org/journals/bul/138/2/322/

On Sat, Jan 9, 2016 at 3:55 PM, Ariel Rokem <arokem@gmail.com> wrote:

> I've asked my colleague for key references on the issue, and I hope he
> will respond with something better, but for the time being here's one
> survey of the literature on this:
> http://www.haskins.yale.edu/sr/SR111/SR111_13.pdf
>
>
> On Sat, Jan 9, 2016 at 2:40 PM, W. Trevor King <notifications@github.com>
> wrote:
>
>> In 01-introduction.md
>> <https://github.com/swcarpentry/instructor-training/pull/61#discussion_r49269009>
>> :
>>
>> > @@ -74,11 +74,11 @@ so that they get the positive feedback of being able to read whole stories as ea
>> >  The basic idea is that once children find reading rewarding,
>> >  they'll be motivated to go back and figure out the letters.
>> >
>> > -Both approaches are consistent with what we know about how children learn,
>> > -so we need to study them in action to determine which is better.
>> > -The findings have been contradictory:
>> > -while some studies find that phonics works best,
>> > -others show that whole language yields better results.
>> > +These approaches are not equally consistent with what we know about how children
>> > +learn. In fact, the scientific consensus today is that the whole word approach
>> > +was a mistake. Nevertheless, when these methods were studied in classrooms, the
>>
>> Links backing up the consensus assertion would be great, especially when
>> followed up by claims of contradictory classroom studies (where links to
>> those studies would also be useful). Hand-waving about consensus/studies
>> without citing references seems un-sciency ;).
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/swcarpentry/instructor-training/pull/61/files#r49269009>
>> .
>>
>
>